requirements.txt

```diff
@@ -1,2 +1,4 @@
 click
 requests
+scrapy
+six
```

Comment:
`six` should be added to `install_requires` in setup.py since it's a hard dependency now.